Web/Form.java

```diff
@@ -0,0 +1,83 @@
+import java.io.IOException;
+import java.io.PrintWriter;
+
+/**
+ * Servlet implementation class Form
+ */
+@WebServlet("/Form")
+public class Form extends HttpServlet {
+    private static final long serialVersionUID = 1L;
+    private String errorMessage = "";
+
+    /**
+     * @see HttpServlet#HttpServlet()
+     */
+    public Form() {
+	super();
+	// TODO Auto-generated constructor stub
+    }
+
+    /**
+     * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
+     */
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+	try {
+	    String name = request.getParameter("name").trim();
+	    String password = request.getParameter("password").trim();
+	    String cPassword = request.getParameter("cPassword").trim();
+	    String email = request.getParameter("email").trim();
+	    String phoneNumber = request.getParameter("pNumber").trim();
+	    response.setContentType("text/html");
+
+	    PrintWriter output = response.getWriter();
+	    if (!verifyData(name, password, cPassword, email, phoneNumber)) {
+		output.println(errorMessage);
+	    } else {
+		RequestDispatcher requestDispatcherObject = request.getRequestDispatcher("SecondServlet");
+		requestDispatcherObject.forward(request, response);
+	    }
+	} catch (NullPointerException exception) {
+	    response.sendRedirect("index.html");
+	}
+    }
+
+    private boolean verifyData(String... data) {
+	boolean returnValue = true;
+	if (data[0].equals("") || data[1].equals("") || data[2].equals("") || data[3].equals("") || data[4].equals("")) {
+
+	    errorMessage = "Form Field can't be empty!!";
+	    returnValue = false;
```

Comment:
You can return from the method when condition break first time. Or you can add all error messages in single message and throw that message at end.